src/black/linegen.py

```diff
@@ -329,7 +329,24 @@ def visit_STRING(self, leaf: Leaf) -> Iterator[Line]:
 
             # We could enforce triple quotes at this point.
             quote = quote_char * quote_len
-            leaf.value = prefix + quote + docstring + quote
+
+            if Preview.long_docstring_quotes_on_newline in self.mode:
+                leaf.value = prefix + quote + docstring + quote
+            else:
```

Comment:
This should be the other way around, no? The idea is that if preview is enabled, we do the new thing.

Reply:
I've fixed this now. Changed a test to have the old incorrect formatting as well

Reply:
Please add the tests to a dedicated file and name that file [here](https://github.com/psf/black/blob/main/tests/test_format.py#L80) so we can test with preview on!

Reply:
Do you want the tests in the new file and the old file?

Reply:
I assume you want the old tests as well so I've kept them in

Reply:
Yeah that works, can't have too many 👍